src/main/java/crawlercommons/sitemaps/sax/XMLHandler.java

```diff
@@ -72,7 +72,20 @@
     public void startElement(String uri, String localName, String qName, Attributes attributes) throws SAXException {
         // flush any unclosed or missing URL element
         if (loc.length() > 0 && ("loc".equals(qName) || "url".equals(qName))) {
-            maybeAddSiteMapUrl();
+            // check whether loc isn't white space only
+            for (int i = 0; i < loc.length(); i++) {
```

Comment:
I thought we already had it but this is not the case apparently. Probably because we get it for free as a subdependency in so many projects that I expect it to be always here. No big deal, let's leave the code as it is.